loops-and-arrays.js

```diff
@@ -42,7 +42,7 @@ for (i = 0; i < numbers.length; i++) {
 
```

Comment:
Opdracht 1 is goed geschreven, echter is de opdracht om een array terug te geven. Omdat je jouw console.log in je loop hebt geschreven, krijg je geen array terug. Zorg ervoor dat jouw console.log dus buiten de loop staat. Alleen het probleem is nu omdat je een loop hebt geschreven en je daar geen if statement aan hebt gegeven, jouw console.log dus niks kan loggen.  Daarom kun je opdracht 1 beter zo schrijven:
`const names = ["Henk", "Piet", "Fred", "Joop"];
console.log(names.join("je ") + "je")`
Deze zal een array terug geven


Opdracht 2 is eigenlijk hetzelfde. Gebruik je console.log buiten de loop en je zult een array terug krijgen.